extension/src/popup/components/hardwareConnect/LedgerConnect/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import TransportWebUSB from "@ledgerhq/hw-transport-webusb";
+import LedgerApi from "@ledgerhq/hw-app-str";
+import { Icon, InfoBlock } from "@stellar/design-system";
+
+import { Button } from "popup/basics/buttons/Button";
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import { SubviewHeader } from "popup/components/SubviewHeader";
+import Ledger from "popup/assets/ledger.png";
+import LedgerConnected from "popup/assets/ledger-connected.png";
+
+import "./styles.scss";
+
+export const defaultStellarBipPath = "44'/148'/0'";
+
+enum LEDGER_ERROR {
+  NO_DEVICE = "NO_DEVICE",
+  NOT_OPEN = "NOT_OPEN",
+  OTHER = "OTHER",
+}
+
+const parseLedgerError = (err: any): string => {
+  const { message } = err;
+  if (!message) {
+    return LEDGER_ERROR.OTHER;
+  }
+  if (message.indexOf("No device selected") > -1) {
+    return LEDGER_ERROR.NO_DEVICE;
+  }
+  if (message.indexOf("Incorrect length")) {
+    return LEDGER_ERROR.NOT_OPEN;
+  }
+  return LEDGER_ERROR.OTHER;
+};
+
+export const LedgerConnect = ({ goBack }: { goBack?: () => void }) => {
+  const [isConnecting, setIsConnecting] = useState(false);
+
+  // TODO - move to redux with bipPath
```

Comment:
will most probably move the connect stuff into redux, wasn't sure of the best API yet so doing in later PR